source/includes/api/catalog/_product.md

```diff
@@ -232,61 +191,31 @@ curl -X "POST" "https://dashboard.shoprunback.com/api/v1/products" \
 
 ```php
 <?php
-
-// Get cURL resource
-$ch = curl_init();
-
-// Set url
-curl_setopt($ch, CURLOPT_URL, 'http://localhost:3000/api/v1/products');
-
-// Set method
-curl_setopt($ch, CURLOPT_CUSTOMREQUEST, 'POST');
-
-// Set options
-curl_setopt($ch, CURLOPT_RETURNTRANSFER, 1);
-
-// Set headers
-curl_setopt($ch, CURLOPT_HTTPHEADER, [
-  "Authorization: Token token=<your token>",
-  "Content-Type: application/json; charset=utf-8",
- ]
-);
-// Create body
-$json_array = [
-            "reference" => "IPHONE 14S B",
-            "id" => "1f27f9d9-3b5c-4152-98b7-760f56967dea",
-            "picture_url" => "https://s3.eu-central-1.amazonaws.com/shoprunback-dev/products/484/7b4/f4-/pictures/medium.?1505912197",
-            "label" => "Iphone 14S Blue",
-            "brand" => [
-                "id" => "20d0be72-a191-4eb8-af22-9dd4c3e65fe8",
-                "name" => "Apple",
-                "reference" => "apple"
-            ],
-            "weight_grams" => 200,
-            "ean" => "1258987561456"
-        ];
-$body = json_encode($json_array);
-
-// Set body
-curl_setopt($ch, CURLOPT_POST, 1);
-curl_setopt($ch, CURLOPT_POSTFIELDS, $body);
-
-// Send the request & save response to $resp
-$resp = curl_exec($ch);
-
-if(!$resp) {
-  die('Error: "' . curl_error($ch) . '" - Code: ' . curl_errno($ch));
-} else {
-  echo "Response HTTP Status Code : " . curl_getinfo($ch, CURLINFO_HTTP_CODE);
-  echo "\nResponse HTTP Body : " . $resp;
-}
-
-// Close request to clear up some resources
-curl_close($ch);
-?>
+// Load the library
+require 'path/to/lib/shoprunback-php/init.php';
+
+// Set your token
+\Shoprunback\RestClient::getClient()->setToken('your_token');
```

Comment:
isn't it needed to set the API url / environment ?

Reply:
It is indeed, I added it just below